pkg/index.yaml

```diff
@@ -300,9 +300,22 @@ entries:
     urls:
     - https://raw.githubusercontent.com/cisco-sso/charts/master/pkg/nginx-0.4.2.tgz
     version: 0.4.2
+  rundeck:
+  - apiVersion: v1
+    appVersion: "1.0"
```

Comment:
Should probably be semver.  e.g. 1.0.0

Reply:
Dave is right.  appVersion is not the chart version though, and must be changed in the chart yaml.

1.x is awhile ago in RunDeck releases http://rundeck.org/docs/history/changelog.html  @Deepak-Routray please correct it to match the version of RunDeck shipped in your container (not the container tag).